app/managers.py

```diff
@@ -0,0 +1,43 @@
+import sqlite3
+from typing import List
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            "(first_name, last_name) VALUES (?,?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> List[Actor]:
+        cinema_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [
+            Actor(*row) for row in cinema_cursor
```

Comment:
Ensure that the `Actor` class constructor in `models.py` matches the order and number of columns returned by the `SELECT *` query. This is crucial for the `all` method to correctly instantiate `Actor` objects from database rows.